Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/remesh.h

```diff
@@ -107,6 +107,12 @@ namespace Polygon_mesh_processing {
 *    constrained in `edge_is_constrained_map` and boundary edges move along the
 *    constrained polylines they belong to.
 *  \cgalParamEnd
+*  \cgalParamBegin{projection_functor}
+*  A function object used to project vertices created. It must have
```

Comment:
It's not only created vertices, but all non-constrained-non-boundary vertices that are projected. The smoothing step moves input vertices, that are also projected by the functor. I suggest : "A function object used to project moving vertices" (this includes input vertices and created vertices)